src/website/app/Heading.js

```diff
@@ -31,6 +31,12 @@ type Props = {
 };
 
 const componentTheme = baseTheme => ({
+  Heading_color_1: baseTheme.color_text,
+  Heading_color_2: baseTheme.color_gray_80,
+  Heading_color_3: baseTheme.color_gray_80,
+  Heading_color_4: baseTheme.color_gray_80,
+  Heading_color_5: baseTheme.color_text,
```

Comment:
This is an odd convention, no? (The `h5` color in particular...) I see that this matches the spec, but can we validate that it is not a mistake?